backend/kernelCI_app/utils.py

```diff
@@ -7,6 +7,16 @@
 DEFAULT_QUERY_TIME_INTERVAL = {"days": 7}
 
 
+def create_issue(issue_id, issue_comment, issue_report_url, incident_id, incident_present):
```

Comment:
```suggestion
def create_issue(*, issue_id, issue_comment, issue_report_url, incident_id, incident_present):
```
So this function only accept keyword parametesr, there are too many arguments